Partial backport of:

commit 6d9f97e1f2b60fe6ca33e0e75e8048c35a39df0e
Author: Siddhesh Poyarekar <siddhesh@redhat.com>
Date:   Fri Dec 28 18:53:17 2012 +0530

    Replace more constants with their values

Only the removal of unused constants from header files is uncluded
here.

diff --git a/sysdeps/ieee754/dbl-64/ulog.h b/sysdeps/ieee754/dbl-64/ulog.h
index 0bca23f0d5d08719..877e747d720ef8fb 100644
--- a/sysdeps/ieee754/dbl-64/ulog.h
+++ b/sysdeps/ieee754/dbl-64/ulog.h
@@ -80,10 +80,6 @@
 /**/ d19            = {{0x3faaf286, 0xbca1af28} }, /*  1/19     */
 /**/ d20            = {{0xbfa99999, 0x9999999a} }, /* -1/20     */
   /* constants    */
-/**/ zero           = {{0x00000000, 0x00000000} }, /* 0         */
-/**/ one            = {{0x3ff00000, 0x00000000} }, /* 1         */
-/**/ half           = {{0x3fe00000, 0x00000000} }, /* 1/2       */
-/**/ mhalf          = {{0xbfe00000, 0x00000000} }, /* -1/2      */
 /**/ sqrt_2         = {{0x3ff6a09e, 0x667f3bcc} }, /* sqrt(2)   */
 /**/ h1             = {{0x3fd2e000, 0x00000000} }, /* 151/2**9  */
 /**/ h2             = {{0x3f669000, 0x00000000} }, /* 361/2**17 */
@@ -156,10 +152,6 @@
 /**/ d19            = {{0xbca1af28, 0x3faaf286} }, /*  1/19     */
 /**/ d20            = {{0x9999999a, 0xbfa99999} }, /* -1/20     */
   /* constants    */
-/**/ zero           = {{0x00000000, 0x00000000} }, /* 0         */
-/**/ one            = {{0x00000000, 0x3ff00000} }, /* 1         */
-/**/ half           = {{0x00000000, 0x3fe00000} }, /* 1/2       */
-/**/ mhalf          = {{0x00000000, 0xbfe00000} }, /* -1/2      */
 /**/ sqrt_2         = {{0x667f3bcc, 0x3ff6a09e} }, /* sqrt(2)   */
 /**/ h1             = {{0x00000000, 0x3fd2e000} }, /* 151/2**9  */
 /**/ h2             = {{0x00000000, 0x3f669000} }, /* 361/2**17 */
diff --git a/sysdeps/ieee754/dbl-64/utan.h b/sysdeps/ieee754/dbl-64/utan.h
index cfbb5f9397e00fec..e7456b71c35fd530 100644
--- a/sysdeps/ieee754/dbl-64/utan.h
+++ b/sysdeps/ieee754/dbl-64/utan.h
@@ -83,9 +83,7 @@
   /* constants    */
 /**/ zero           = {{0x00000000, 0x00000000} }, /* 0         */
 /**/ one            = {{0x3ff00000, 0x00000000} }, /* 1         */
-/**/ mone           = {{0xbff00000, 0x00000000} }, /*-1         */
 /**/ mfftnhf        = {{0xc02f0000, 0x00000000} }, /*-15.5      */
-/**/ two8           = {{0x40700000, 0x00000000} }, /* 256       */
 
 /**/ g1             = {{0x3e4b096c, 0x00000000} }, /* 1.259e-8  */
 /**/ g2             = {{0x3faf212d, 0x00000000} }, /* 0.0608    */
@@ -204,9 +202,7 @@
   /* constants    */
 /**/ zero           = {{0x00000000, 0x00000000} }, /* 0         */
 /**/ one            = {{0x00000000, 0x3ff00000} }, /* 1         */
-/**/ mone           = {{0x00000000, 0xbff00000} }, /*-1         */
 /**/ mfftnhf        = {{0x00000000, 0xc02f0000} }, /*-15.5      */
-/**/ two8           = {{0x00000000, 0x40700000} }, /* 256       */
 
 /**/ g1             = {{0x00000000, 0x3e4b096c} }, /* 1.259e-8  */
 /**/ g2             = {{0x00000000, 0x3faf212d} }, /* 0.0608    */
